Dockerfile

```
@@ -1,12 +1,30 @@
+# Build JS app
+FROM node:current-alpine AS builder
```

instead of auto updating to latest node, lets stick with the currently last supported lts version: 12.

```suggestion
FROM 12-alpine AS builder
```